csstools/index.html

```diff
@@ -786,7 +786,34 @@
      </section>
     </div>
     
-   
+
+    <div class="slide section"> 
+     <header><h1>Extras</h1></header> 
+     <section class="content"></section>
+    </div>
+    
+    <div class="slide normal"> 
+      <header><h1>Other things worth considering</h1></header> 
+      <section class="content"> 
+        <p>Performance</p>
+        <ul>
+          <li>CSS file size and compression
+          <li>Image sprites
+          <li>Data URI
+          <li>Icon fonts
```

Comment:
Maybe:
Improving rendering speed with CSS:
- Image sprites (with a link to an article)
- Data URIs ( with a link to an article)
- Icon fonts (with a link to an article)
Probably CSStricks articles or whatever is top Google result